Dockerfile.template

```
@@ -69,7 +69,6 @@ RUN set -eux; \
 {{ if is_alpine then ( -}}
 	apk add --no-cache --virtual .ruby-builddeps \
 		autoconf \
-		bison \
 		bzip2 \
```

This should match what we do for `readline-dev`: :eyes:

```suggestion
{{ if [ "3.0", "3.1", "3.2" ] | index(env.version | rtrimstr("-rc")) then ( -}}
{{ # https://github.com/docker-library/ruby/pull/438 -}}
		bison \
{{ ) else "" end -}}
		bzip2 \
```